docs/src/examples/dimetourrotations.md

```diff
@@ -0,0 +1,144 @@
+```
+
+Now let's say we want to define a manifold point `i` some rotation θ from the [`identity`](@ref) reference `xR0`.  This is easier to envision on `Rotations(2)`, while more complicated would likely use a generalized notion of distance between points instead.  For now considering a rotation, say
+```julia
+# + radians rotation from x-axis on plane to point i
+xθi = π/6
+```
+
+### From Coordinates
+
+To get our first Lie algebra element using the text book [`hat`](@ref), or equivaliently a more generalized [`get_vector`](@ref), function:
+```julia
+X_ = hat(G, xR0, xθi)             # specific definition
+xXi = get_vector(G, xR0, xθi, e0)  # generalized definition
+# 2×2 MMatrix{2, 2, Float64, 4} with indices SOneTo(2)×SOneTo(2):
+#  0.0       -0.523599
+#  0.523599   0.0
+@assert isapprox( X_, xXi )
+```
+
+Note, in this case the same would work given the base manifold [`Rotations(2)`](@ref):
+```julia
+_X_ = hat(M, xR0, xθi)             # specific definition
+_X = get_vector(M, xR0, xθi, e0)  # generalized definition
+@assert _X_ == xXi; @assert _X == xXi
+```
+
+Now, let's place this algebra element on the manifold using the exponential map [`exp`](@ref):
+```julia
+xRi = exp(G, xR0, xXi)
+# similarly for known underlying manifold
+xRi_ = exp(M, xR0, xXi)
+
+@assert isapprox( xRi, xRi_ )
+```
+
+### To Coordinates
+
+The logarithm transform from the group back to algebra (or coordinates) is:
+```julia
+xXi_ = log(G, xR0, xRi)
+xXi__ = log(M, xR0, xRi)
+@assert xXi == xXi__
+```
+
+Similarly, the coordinate value can be extracted from the algebra using [`vee`](@ref), or directly from the group using the more generalized [`get_coordinates`](@ref):
+```julia
+# extracting coordinates using vee
+xθi__ = vee(G, xR0, xXi_)[1]
+_xθi__ = vee(M, xR0, xXi_)[1]
+
+# OR, the preferred generalized get_coordinate function
+xθi_ = get_coordinates(G, xR0, xXi_, e0)[1]
+_xθi_ = get_coordinates(M, xR0, xXi_, e0)[1]
+
+# confirm all versions are correct
+@assert isapprox( xθi, xθi_ ); @assert isapprox( xθi, _xθi_ )
+@assert isapprox( xθi, xθi__ ); @assert isapprox( xθi, _xθi__ )
+```
+
+### Actions and Operations
+
+With the basics in hand on how to move between the coordinate, algebra, and group representations, let's briefly look at composition and application of points on the manifold.  For example, a `Rotation(n)` manifold is the mathematical representation, but the points have an application purpose in retaining information regarding a specific rotation.  In contrast, using Euler angles, or Euclidean(n) space to store rotation information quickly becomes problematic.  Other rotation representation methods, including quaternions, Pauli matrices, etc., have similar features.
+
+Therefore, points from a manifold may have an associated action which we [`apply`](@ref).  Consider rotating through `θ = π/6` three vectors `V` from their native domain `Euclidean(2)`, from the reference frame `a` to frame `b`.  Keeping with our two-dimensional example above:
```

Comment:
```suggestion
Therefore, points from a Lie group may have an associated action which we [`apply`](@ref).  Consider rotating through `θ = π/6` three vectors `V` from their native domain `Euclidean(2)`, from the reference frame `a` to frame `b`.  Keeping with our two-dimensional example above:
```
Why is there a "therefore"?

I'm not sure why you mentioned reference frames here? I think there may be some confusion regarding active vs passive transformation. Our actions are, IIRC, always active currently. A passive action would have to work with bases of tangent spaces, since you can't even define reference frames on general manifolds. This also somewhat translates to the euclidean case where you can select origin of your reference frame arbitrarily.

Reply:
fixed